<launch>
	<!-- Generic parameters: base_link -->
	<param name="hostname" value="192.168.1.3" />

	<!-- Generic parameters: Grid map -->
	<param name="grid_x" value="2.0"  type="double" />
	<param name="grid_y" value="2.0"  type="double" />
	<param name="grid_r" value="0.02" type="double" />
	<param name="grid_frame" value="base_link" />

	<!-- Generic parameters: nodes -->
	<param name="frequency" value="10.0" />

	<!-- Robotino base_link node -->
	<node name="robotino_node_base" pkg="cnbiros_robotino" type="robotino_node_base" output="screen" >
		<param name="topic" value="/command_velocity" />
	</node>
   
	<!-- Robotino infrared node -->
	<node name="robotino_node_infrared" pkg="cnbiros_robotino" type="robotino_node_infrared" output="screen" >
		<param name="topic" value="/sensor_infrared" />
		<param name="grid_layer" value="infrared" />
		<param name="decay" value="1" type="double"/>
	</node>
   
	<!-- Robotino fusion node -->
	<node name="robotino_node_fusion" pkg="cnbiros_robotino" type="robotino_node_fusion" output="screen" >
		<param name="topic" value="/sensors_fusion" />
		<param name="grid_layer" value="fusion" />
	</node>
	
	<!-- Launch rviz -->
	<node name="rviz" pkg="rviz" type="rviz" args="-d $(find cnbiros_robotino)/rviz/grid_map.rviz" />
</launch>
